plugins/dell-dock2/fu-dell-dock2-ilan-firmware.c

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2024 Dell Technologies
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR MIT
+ */
+
+#include "config.h"
+
+#include "fu-dell-dock2-common.h"
+#include "fu-dell-dock2-ilan-firmware.h"
+
+#define DOCK_ILAN_VERSION_OFFSET 0x0A
+
+struct _FuDellDock2IlanFirmware {
+	FuFirmwareClass parent_instance;
+};
+
+G_DEFINE_TYPE(FuDellDock2IlanFirmware, fu_dell_dock2_ilan_firmware, FU_TYPE_FIRMWARE)
+
+static gboolean
+fu_dell_dock2_ilan_firmware_parse(FuFirmware *firmware,
+				  GInputStream *stream,
+				  gsize offset,
+				  FwupdInstallFlags flags,
+				  GError **error)
+{
+	guint16 version_raw = 0;
+	guint8 version_major = 0;
+	guint8 version_minor = 0;
```

Comment:
these two don't need to be initialized, they are written to every time before use.

Reply:
Fixed, thanks.